src/Orckestra.Composer.Cart/Providers/Order/EditingOrderProvider.cs

```diff
@@ -98,14 +101,12 @@ private bool HasCancellationMessage(OrderFulfillmentState orderFulfillmentState)
 
         private bool ValidateOrderForCancel(Overture.ServiceModel.Orders.Order order)
```

Comment:
please rename this method to ValidateOrder(order) and use it in CanEdit also. It looks like use same logic